lib/l10n/intl_de.arb

```diff
@@ -28,6 +28,21 @@
   "@app_errorScreen_unknown_showStackTrace": {
     "type": "text"
   },
+  "app_logOut_confirm": "Abmelden",
+  "@app_logOut_confirm": {
+    "type": "text",
+    "description": ""
+  },
+  "app_logOut_content": "Du brauchst deine Anmeldedaten, um dich wieder anzumelden.",
```

Comment:
Maybe change this something more clear like "Du musst dich danach erneut anmelden, um die App zu benutzen"

Reply:
Klingt sinnvoll